.github/workflows/mac.yml

```diff
@@ -107,12 +107,109 @@ jobs:
       shell: bash
       run: |
         set -e -x
-        python3 ${{ github.workspace }}/tools/ci_build/build.py --test --build_objc --build_wheel --use_xnnpack --build_nodejs --build_java --use_coreml --use_webgpu --build_dir ${{ github.workspace }} --skip_submodule_sync --parallel --use_binskim_compliant_compile_flags --build_shared_lib --config Release --use_vcpkg --use_vcpkg_ms_internal_asset_cache
+        python3 ${{ github.workspace }}/tools/ci_build/build.py --test --build_objc --build_wheel --use_xnnpack --build_nodejs --build_java --use_coreml --build_dir ${{ github.workspace }} --skip_submodule_sync --parallel --use_binskim_compliant_compile_flags --build_shared_lib --config Release --use_vcpkg --use_vcpkg_ms_internal_asset_cache
     timeout-minutes: 300
     env:
-      MACOSX_DEPLOYMENT_TARGET: '13.3'
-      ALLOW_RELEASED_ONNX_OPSET_ONLY: '0'
+      MACOSX_DEPLOYMENT_TARGET: "13.3"
+      ALLOW_RELEASED_ONNX_OPSET_ONLY: "0"
 
+  x86_64-Xcode16_webgpu:
+    runs-on: macos-15
```

Comment:
I think the "macos-15" runner is an arm64 machine.
https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

will this job actually build and test x86_64 code?

Reply:
thank you. i didn't know that. Let me check

Reply:
reverted to macOS 13 for x86_64.